frontend/docs/character.md

```diff
@@ -118,4 +118,78 @@ https://finalspaceapi.com/api/v0/character/?sort=desc
 ]
 ```
 
+### Limit numbers of returend characters
```

Comment:
Spelling error returned
You can make it `Limit the response`

Reply:
ok 👍🏼